build-tools/installers/create-installers.targets

```diff
@@ -137,18 +156,29 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\libZipSharp.dll.config" Condition=" '$(HostOS)' != 'Windows' " />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Posix.NETStandard.dll" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Profiler.Log.dll" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Profiler.Log.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\logcat-parse.exe" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\logcat-parse.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\mdoc.exe" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\mdoc.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle.exe" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle-api.h" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\mono-symbolicate.exe" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\mono-symbolicate.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.CSharp.dll" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.CSharp.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.CompilerServices.SymbolWriter.dll" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.CompilerServices.SymbolWriter.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Options.dll" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Options.pdb" />
```

Comment:
Alas, this doesn't exist, because the [Mono.Options NuGet package](https://www.nuget.org/packages/Mono.Options/) doesn't contain `.pdb` files.